rmgpy/cantherm/qchem.py

```diff
@@ -115,19 +115,30 @@ def loadGeometry(self):
         """
         atom = []; coord = []; number = []; 
 
+        #First check that the Qchem job file (not necessarily a geometry optimization)
+        #has successfully completed, if not an error is thrown
+        completedjob = 0;
         f = open(self.path, 'r')
         line = f.readline()
         while line != '':
-            if 'Final energy is' in line:
-                print 'found a sucessfully completed Qchem Geometry Optimization Job'
+            if 'Total job time:' in line:
+                print('Found a sucessfully completed Qchem Job')
```

Comment:
I think that the `print` here should be `logging.debug` instead (although originally it was a print statement as well)